src/Ide/Plugin/Ormolu.hs

```diff
@@ -51,18 +47,7 @@ descriptor plId = PluginDescriptor
 
 provider :: FormattingProvider IO
 #if __GLASGOW_HASKELL__ >= 806
```

Comment:
For future, we are not building below 8.6, can remove this. But should probably put a constraint on base in the main cabal file to enforce that.